ui/src/components/ClubsCarousel/index.tsx

```diff
@@ -2,16 +2,40 @@ import styles from './styles.module.scss';
 import Carousel from 'react-multi-carousel';
 import 'react-multi-carousel/lib/styles.css';
 import {useAuthFetch} from '../../api/useAuthFetch';
-import {useEffect} from 'react';
+import {useEffect, useState} from 'react';
 import {useLogout} from '../../hooks/useLogout';
+import {HeadingDropdown} from '../HeadingDropdown';
 
 interface ClubsCarouselData {
   name: string;
   logo: string;
   category: string;
 }
 
+const responsive = {
```

Comment:
Let's use [Immutable](https://www.npmjs.com/package/immutable) for constants. This would prevent people from mistakenly updating the constant values.

Feel free to use any other library which prevents updation of constants because js/ts doesn't do it.